impy/impy_config.yaml

```diff
@@ -104,7 +104,6 @@ urqmd:
     # at plab < 4.9 use euation of state and Lorentz contraction of projectile
     # and target. This is not set necessarily in CORSIKA
     LE_EOS       : True
-    debug_level  : 0
```

Comment:
This one sets a separate [debug level for URQMD in FORTRAN](https://github.com/impy-project/impy/blob/44e548d69c240c9c37c1d8a6e2314e1dba578187/impy/models/urqmd.py#L178). It's in a subdictionary, so shouldn't generate a name clash. It might be better to simply change this to urqmd_debug_level or so to get a clean yaml check.

Reply:
There are two debug_level defined in the same dict, this is an error, please look again.

Reply:
The linters I recommend to enable do not generate false positives in my experience, so it is safe to apply them. There are some linters which do, and we should keep away from those. They are causing more harm than good.

Reply:
OK! Gotcha. Oversight. Actually all generators have that flag ;)
I reverted my change. The PR is good to go once #20 is merged.